src/lib/components/Pagination/Pagination.js

```diff
@@ -32,8 +32,8 @@ class Pagination extends Component {
           <Paginator
             activePage={currentPage}
             totalPages={totalDisplayedPages}
-            firstItem={simple ? null : firstItem}
-            lastItem={simple ? null : lastItem}
+            firstItem={null}
```

Comment:
With this change the `simple` prop is not supported any longer, we should remove it from the `PropTypes` and `defaultProps` and also from the components 

- `ExistingRelations.js`
- `frontsite/PatronProfile/LoanList.js`
- `PatronCurrentDocumentRequests.js`
- `PatronPastDocumentRequests.js`

Reply:
Yes, I'm the one who added it in the first place; I was aware of that and left it voluntarily in case we want to have specific logic for smaller screens. I'll remove it if no one is against.